### PR TITLE
Update WPF binary names with _cor3 suffix to _net6

### PR DESCRIPTION
**Pull request by ryalanms**
PenImc_cor3.dll, wpfgfx_cor3.dll, and PresentationNative_cor3.dll were named during the port from .NET Framework to .NET Core and need to be updated.

These have been updated to _net6 from _cor3.  Fixes https://github.com/dotnet/wpf/issues/4062.  

/cc @SamBent

### CODE REVIEW

File: eng/WpfArcadeSdk/tools/WpfProjectReference.targets [hunk 1]
```diff
@@ -54,19 +54,19 @@
     <WpfProjectPath Include="UIAutomationClientSideProviders" ProjectPath="$(WpfSourceDir)UIAutomation\UIAutomationClientSideProviders\UIAutomationClientSideProviders.csproj" />
 
 
-    <WpfProjectPath Include="PresentationNative_cor3" ProjectPath="$(WpfSourceDir)PresentationNative\dll\PresentationNative.vcxproj">
+    <WpfProjectPath Include="PresentationNative_net6" ProjectPath="$(WpfSourceDir)PresentationNative\dll\PresentationNative.vcxproj">
```
Comment (batzen) [inline eng/WpfArcadeSdk/tools/WpfProjectReference.targets:57]: Shouldn't these includes use the suffix property to build the name?

**Comment by ryalanms** [comment]
This is an update to the Wpf Arcade SDK which will require breaking the build.  wpf-int must consume the new SDK and push a package before public can build successfully.

**Comment by ThomasGoulet73** [comment]
Shouldn't you also update this file: https://github.com/dotnet/wpf/blob/d49f8ddb889b5717437d03caa04d7c56819c16aa/src/Microsoft.DotNet.Wpf/src/Shared/MS/Win32/ExternDll.cs#L36 ? It seems to be used in code for the DllImportAttribute.

File: eng/WpfArcadeSdk/tools/WpfProjectReference.targets (hunk 1 above)
Comment (ryalanms) [reply eng/WpfArcadeSdk/tools/WpfProjectReference.targets]: Thanks, @batzen.

**Comment by Nirmal4G** [comment]
Do we really need to specify a version or even a suffix at all?

**Comment by dipeshmsft** [comment]
Closing this PR as stale.

**Comment by batzen** [comment]
Stale? The reviews from the WPF team didn't even happen.
There was no further feedback the author could have acted on.

**Pull request closed without merging**